server/game/cards/11-TCaR/DevilsSixGun.js

```diff
@@ -0,0 +1,62 @@
+const GoodsCard = require('../../goodscard.js');
+const GameActions = require('../../GameActions/index.js');
+let selectedJoker = null;
```

Comment:
Should this variable be pushed down into the action? It isn't used outside of the action.

Reply:
Yes, if you want to use variable in cards across functions, it is better to use `this.variable`. However, I have reworked the discard event and updated this PR so the variable is no longer needed.